src/Core/Content/Media/Extension/ResolveRemoteThumbnailUrlExtension.php

```diff
@@ -20,12 +23,11 @@ final class ResolveRemoteThumbnailUrlExtension extends Extension
      * @internal shopware owns the __constructor, but the properties are public API
      */
     public function __construct(
+        public PartialEntity|MediaEntity $mediaEntity,
         public string $mediaUrl,
-        public string $mediaPath,
         public string $width,
         public string $height,
-        public string $pattern,
-        public ?\DateTimeInterface $mediaUpdatedAt
```

Comment:
adding new properties is okay, but you should not remove existing ones. you can deprecate them, if you think, they are obsolete